bipole5/BipoleV.py

```diff
@@ -749,29 +1310,39 @@ def disable_keys():
     left_command = "nothing()"
     right_command = "nothing()"
 
-dialogue = ["install","bipole"]
+
+dialogue = ["install", "bipole"]
 dialogue_index = 1
 
+
 def start_dialogue(input_file):
     disable_inputs()
     global dialogue
     global dialogue_index
     dialogue_index = 1
-    file_contents = open(current_directory+"/dialogue/"+maps.current_location[4]+"/"+input_file+".txt")
-    dialogue = [line.rstrip('\n') for line in file_contents]
+    file_contents = open(
+        current_directory
+        / "dialogue"
+        / maps.current_location[4]
+        / (input_file + ".txt"),
+        encoding="UTF-8",
+    )
```

Comment:
Use the '/' operator to append strings to a path, @infinityJKA.
Oh, and when opening in text ("t") mode, always specify an encoding.